test/Interpreter/errors.swift

```diff
@@ -0,0 +1,72 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+//
+// Tests for error handling.
+//
+
+import StdlibUnittest
+
+// Also import modules which are used by StdlibUnittest internally. This
+// workaround is needed to link all required libraries in case we compile
+// StdlibUnittest with -sil-serialize-all.
+import SwiftPrivate
+#if _runtime(_ObjC)
+import ObjectiveC
+#endif
+
+var count = 0
+
+enum Markings { case Tuxedo, Tortoise }
+
+class Cat : Equatable {
```

Comment:
I like cats, but StdlibUnittest provides `LifetimeTracked` type for leak checking.  And the harness automatically checks that `LifetimeTracked.instances == 0` at the end of every test.